### ISSUE TITLE
Increase the token scope size limit from 2048

### ISSUE THREAD

**Issue by jgeschwendt** [markhuot/craftql#139]
I have a project that has over 25 section types. Any time I save the JSON string gets truncated and causes that token to fail.

**Comment by markhuot** [comment]
Interesting, thanks for the heads up @jgeschwendt. I'm going to move this over to a `text` database column in the next release. Feel free to make the update to your MySQL database manually in the interim.

**Comment by bovasso** [comment]
Having the same issue. Text would be ideal.

**Comment by markhuot** [comment]
Yup, it'll be `text` with the next release. Feel free to update your DB manually in the meantime.
